### ISSUE TITLE
Blocklist Cleanup

### ISSUE THREAD

**Issue by AyanokojiK1yotaka** [WaLLy3K/wally3k.github.io#202]
[i] Target: https://v.firebog.net/hosts/Prigent-Adult.txt
  [✓] Status: Retrieval successful
  [✓] Parsed 4513284 exact domains and 0 ABP-style domains (ignored 4 non-domain entries)
      Sample of non-domain entries:
        - "xxx"
        - "porn"
        - "sex"
        - "adult"

**Comment by KnightmareVIIVIIXC** [comment]
Those look like TLDs

**Comment by AyanokojiK1yotaka** [comment]
yeah i think they are TLDs too but unfortunately there are not being properly implemented for Pihole

**Comment by WaLLy3K** [comment]
Looks like they're the four last entries of that list. I've [submitted a whitelist request for them here](https://dsi.ut-capitole.fr/cgi-bin/squidguard_modify.cgi). Dunno if they'll action the request, but it's no big deal since Pi-hole just rejects these. 🙂
